### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xdoan**
updating readme to streamline and put notes at the end of each section. 

Also @sujaypatil96 not sure if instructions for how to get the token are necessary, its pretty self-explanatory to me, has it been an issue with others?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,48 +12,53 @@ Create a conda environment in the cloned directory from the `environment.yml` fi
...
+### Data Ingress Pipeline Setup (backend)
 
-1. Clone the repo from this [location](https://github.com/Sage-Bionetworks/HTAN-data-pipeline/tree/organized-into-packages), by running the following command:
+The next step is to install the latest release of the [Data Ingress Pipeline](https://github.com/Sage-Bionetworks/HTAN-data-pipeline/tree/organized-into-packages) (backend) as the folder `HTAN-data-pipeline` inside of the `HTAN_data_curator folder` and tie it together with this frontend. To do so carry out the following steps:
+
+1. Inside the HTAN_data_curator folder, clone the repo from this [location](https://github.com/Sage-Bionetworks/HTAN-data-pipeline/tree/organized-into-packages), by running the following command:
 
     `git clone --single-branch --branch organized-into-packages https://github.com/Sage-Bionetworks/HTAN-data-pipeline.git`
 
+This creates a folder named `HTAN-data-pipeline` inside the the `HTAN_data_curator folder` .
```
Comment (sujaypatil96) [inline README.md:34]: Trivial note regarding spacing between `HTAN_data_curator` and period.

File: README.md [hunk 2]
```diff
@@ -11,48 +12,53 @@ Create a conda environment in the cloned directory from the `environment.yml` fi
...
 
-Here, our conda environment name `data_curator_env` is set from the `environment.yml` file . You can change it anything you like, but please note that you will need to make changes accordingly in the `app.R` file.
+Here, our conda environment name `data_curator_env` is set from the `environment.yml` file .
 
 Activate the `data_curator_env` environment:
 
     conda activate data_curator_env
+    
+_Note_:
+- You can change the name of your conda environment inside `environment.yml` or even use another environment, but please note that you will need to make changes accordingly in the `app.R` file line 16.
```
Comment (sujaypatil96) [inline README.md:22]: ..make changes accordingly in the `app.R` file on [line #16](https://github.com/Sage-Bionetworks/HTAN_data_curator/blob/shiny-server-packaged-backend/app.R#L16).

Comment (xdoan) [reply README.md:22]: I don't often use hyperlinks, would we have to manually update this often? e.g. if we move to shiny server or did other updates?

**Comment by xdoan** [comment]
This is in regards to the instructions after the token.pickle like the ones that explain Google's auth stuff, knowing where the token gets downloaded to is useful added info for me personally but the Google UI was self explanatory. 

> This will prompt you with a URL on your console. Copy and paste the URL in your browser. Use your email associated with the Google account to go through the authentication process and allow the application/script (called _Quickstart_) to access Drive/Sheets/etc. You will receive an authentication/authorization code, copy and paste the code in the console. The `token.pickle` file will automatically be downloaded to the required location after that.

Just a thought bc it helps to narrow our scope for skimming when people look over the readme

File: README.md (hunk 2 above)
Comment (sujaypatil96) [reply README.md:22]: Yes, we would have to update this manually, if ever we decide to import a few more/less libraries and the line that imports the conda environment gets shifted. But I guess that makes sense, it's more manual work for us to update. We don't need to have a hyperlink to the line on GitHub. Feel free to resolve the conversation!

**Comment by sujaypatil96** [comment]
@xdoan right that makes sense, the shorter/more concise, the better. The console/browser instructions from Google are pretty clear and should suffice. Feel free to merge the PR!

File: README.md (hunk 2 above)
Comment (xdoan) [reply README.md:22]: if we get to a stable readme on the shiny-server we could add that in or the code example (as long as we don't think it'll change much)

**Pull request merged**